mattermost/mattermost-webapp
# MM-20334 add E2E for "Can change Privacy Policy Link setting"

Summary
Add e2e to the ability to change "Can change Privacy Policy Link setting"

Test steps:
1. Login as a sysadmin and visit the page "/admin_console/site_config/customization" directly
2. Verify that the setting, visible and match the text
3. Verify that the help text, visible and match the text
4. Verify that the input field visible, update its value
5. Verify that the value is saved, directly via REST API

Ticket Link
Jira ticket MM-20334
closes https://github.com/mattermost/mattermost-server/issues/13168

Related Pull Requests
Related PR - #4210
